### ISSUE TITLE
Section Formatting and Descriptions In the Configuration Cell Were Overwritten on Main After PR #2

### ISSUE THREAD

**Issue by Frank-Duffy** [SMB-Lab/FRETNet-Designer#3]
The formatted section in cell 1 on the main branch notebook needs to be restored to the original updated format.

The section formatting and descriptions in the configuration cell were overwritten on the main branch after PR #2.  Why did this happen?

When I went to see if I could recover the section formatting from a previous branch which I knew would have that formatting, I was unable to open the notebook in jupyter due to error "Unable to read notebook: ..... Notebook does not appear to be JSON." I opened the notebook on feature-auto_MSA_generation and found that there were merge conflicts that needed resolving. I accepted all current changes which got the section formatting back the way it should be.

**Comment by Frank-Duffy** [comment]
After testing changes introduced in PR #2 on the associated feature branch, I decided to fix this formatting issue. I created a new branch nbdime-mergetool-test off of FT_Fix_OurSide to attempt to merge the enhanced format from feature-auto_MSA_generation using a nifty jupyter notebook merge tool I found online, nbdime. I could have done this directly on FT_Fix_OurSide but I wanted to try it on a copy just in case I made a fatal error.

When I attempted to merge feature-auto_MSA_generation into nbdime-mergetool-test, I was correctly notified by git that there was a merge conflict in the master notebook. I attempted to resolve by executing nbdime mergetool on the cmd line, but when the mergetool web-app opened up I was greeted by an error "422 unrecognized file." 

After doing some digging I recognized that this could be related an issue with either the remote, local, or base commits. the nbdime mergetool gave me a hint, outputting " Notebook does not appear to be JSON: '{\n "cells": [\n  {\n   "cell_type": "c... ". I've seen that error before. I got it when when I was trying to open a jupyter notebook that wasn't in JSON format. Coincidentally (but it turns out later that it wasn't a coincidence at all..) the last time I got that error was when I was dealing with a merge conflict on the notebook. The <<<<<<< HEAD and >>>>>>> origin lines inserted into the JSON broke the JSON format, making it unreadable by the jupyter server.

So I decided to see whether the notebook in the remote, local or base commits were unreadable, since an issue with any one of them could potentially cause this error. I checked that the remote and local commits were readable by simply switching to each branch and attempting to read the jupyter notebook on the jupyter server. They were readable. I then used git merge-base to get the commit of the base a.k.a common ancestor of the two branches and found that the base was the very commit containing the broken JSON notebook that I had trouble reading before.

So it makes sense that nbdime's web-app wouldn't be able to show the notebook for comparison then, the JSON for the notebook in the is was broken.
